lyber-core.gemspec

```diff
@@ -1,11 +1,10 @@
 # frozen_string_literal: true
 
-lib = File.expand_path('lib', __dir__)
-$:.unshift lib unless $:.include?(lib)
+require_relative "lib/lyber_core/version"
 
 Gem::Specification.new do |s|
   s.name        = 'lyber-core'
-  s.version     = '6.1.1'
+  s.version     = LyberCore::VERSION
   s.licenses    = ['Apache-2.0']
   s.platform    = Gem::Platform::RUBY
   s.authors     = ['Alpana Pande', 'Bess Sadler', 'Chris Fitzpatrick', 'Douglas Kim', 'Richard Anderson', 'Willy Mene', 'Michael Klein', 'Darren Weber', 'Peter Mangiafico']
```

Comment:
aside from Peter ...everyone else is long gone.  Maybe we should start using dlss-infrastructure email.